### PR TITLE
include actions to java-gen-compilation-tests

### PR DESCRIPTION
**Pull request by octonato**

### CODE REVIEW

File: codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/java/SourceGenerator.scala [hunk 1]
```diff
@@ -253,20 +253,10 @@ object SourceGenerator extends PrettyPrinter {
...
-      if (serv.fqn.parent.javaPackage != mainClassPackageName) {
-        val outerClass = s"${serv.fqn.parent.javaPackage}.${serv.fqn.parent.javaOuterClassname}"
-        if (viewWithTransforms)
-          List(serv.fqn.fullName, outerClass)
-        else
-          List(outerClass)
-      } else List.empty
+      if (serv.fqn.parent.javaPackage != mainClassPackageName)
+        serv.fqn.fullName :: s"${serv.fqn.parent.javaPackage}.${serv.fqn.parent.javaOuterClassname}" :: Nil
+      else List.empty
```
Comment (octonato) [inline codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/java/SourceGenerator.scala:258]: This changed in #218, but now reverted to what we have in `main`. Not clear why we added a flag  (`viewWithTransforms`). 

In any case, we do need to include `serv.fqn.fullName` in the imports for the Actions.

File: codegen/core/src/main/scala/com/lightbend/akkasls/codegen/Packaging.scala [hunk 2]
```diff
@@ -58,9 +58,15 @@ object FullyQualifiedName {
...
       serviceType match {
+        // View and Actions services clashes with akka-grpc service generation
+        // therefore we need to append [View,Action] to it (or Impl).
+        // FIXME: should we find a better solution for that?
         case ServiceType.SERVICE_TYPE_VIEW =>
           if (descriptor.getName.endsWith("View")) descriptor.getName + "Impl"
           else descriptor.getName + "View"
+        case ServiceType.SERVICE_TYPE_ACTION =>
+          if (descriptor.getName.endsWith("Action")) descriptor.getName + "Impl"
+          else descriptor.getName + "Action"
```
Comment (octonato) [inline codegen/core/src/main/scala/com/lightbend/akkasls/codegen/Packaging.scala:69]: This is kind of weird and can be surprising for users. I will create a ticket for this. We may want to revisit it and improve later.

File: codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/java/SourceGenerator.scala (hunk 1 above)
Comment (octonato) [reply codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/java/SourceGenerator.scala]: Got to revert this. Tests were failing. 🤷

File: codegen/core/src/main/scala/com/lightbend/akkasls/codegen/Packaging.scala (hunk 2 above)
Comment (patriknw) [reply codegen/core/src/main/scala/com/lightbend/akkasls/codegen/Packaging.scala:69]: it was added in https://github.com/lightbend/akkaserverless-java-sdk/pull/208 (but only for views)

**Review by patriknw — APPROVED** [review]
LGTM

**Review by johanandren — APPROVED** [review]

**Pull request merged**